mysql/plan.sh

```diff
@@ -45,6 +45,12 @@ pkg_bin_dirs=(bin)
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
 
+pkg_exports=(
+    [port]=port
+    [pwd]=app_password
```

Comment:
Why do these need to be abbreviated like this? Why not "password" and "username"?